output_5.1.txt

```diff
@@ -0,0 +1,70 @@
+TASK: 4.2
+
+db.restaurants.createIndex({"name": "Glorious Food"})
+
+{
+        "ok" : 0,
+        "errmsg" : "Unknown index plugin 'Glorious Food'",
+        "code" : 67,
+        "codeName" : "CannotCreateIndex"
+}
```

Comment:
completely incorrect query - as you can see, the result shows you an error: `Unknown index plugin 'Glorious Food'`